pkg/util/utils_windows.go

```diff
@@ -8,24 +8,24 @@ import (
 
 // IsCgroup2UnifiedMode returns whether we are running in cgroup 2 unified mode.
 func IsCgroup2UnifiedMode() (bool, error) {
-	return false, errors.New("this function is not implemented for windows")
+	return false, errors.New("1this function is not implemented for windows")
 }
 
 // GetContainerPidInformationDescriptors returns a string slice of all supported
 // format descriptors of GetContainerPidInformation.
 func GetContainerPidInformationDescriptors() ([]string, error) {
-	return nil, errors.New("this function is not implemented for windows")
+	return nil, errors.New("2this function is not implemented for windows")
 }
 
 // GetRootlessPauseProcessPidPath returns the path to the file that holds the pid for
 // the pause process
 func GetRootlessPauseProcessPidPath() (string, error) {
-	return "", errors.New("this function is not implemented for windows")
+	return "", errors.New("3this function is not implemented for windows")
 }
 
 // GetRootlessRuntimeDir returns the runtime directory when running as non root
 func GetRootlessRuntimeDir() (string, error) {
-	return "", errors.New("this function is not implemented for windows")
+	return "", errors.New("4this function is not implemented for windows")
```

Comment:
I know they'll rarely be seen, but the number before "this" is odd and looks like a type.  I'd much prefer "The GetRootlessRuntimeDir function is not..." if you want to differentiate.

Reply:
We should be sharing the error message if possible - wrap `define.ErrNotImplemented`

Reply:
the numbers are left over debug ... one of the issues with this is that you dont know which error actually returned.

Reply:
Wrap `define.ErrNotImplemented` with the name of the function?

Reply:
used a local errnotimplemented and wrapped it with the function name